vic/drivers/shared/include/vic_driver_shared.h

```diff
@@ -445,6 +445,8 @@ void collect_eb_terms(energy_bal_struct, snow_data_struct, cell_data_struct,
 void collect_wb_terms(cell_data_struct, veg_var_struct, snow_data_struct,
                       double, double, double, int, double, int, double *,
                       double *, out_data_struct *);
+void compute_cell_area(soil_con_struct *);
```

Comment:
Can we put this in the classic driver.  I don't think we need/want it in the image driver(s).